app/controllers/storage_locations_controller.rb

```diff
@@ -11,7 +11,7 @@ def quantity
   def index
     @inventory = View::Inventory.new(current_organization.id)
     @selected_item_category = filter_params[:containing]
-    @items = StorageLocation.items_inventoried(current_organization, @inventory)
```

Comment:
Moved this method, as it doesn't really have anything to do with storage locations.